mscore/mu3launcher.cpp

```diff
@@ -0,0 +1,42 @@
+#include "mu3launcher.h"
+
+#include <QAction>
+
+#include "log.h"
+#include "shortcut.h"
+#include "measureproperties.h"
+
+RetVal<Val> MU3Launcher::open(const std::string& uri)
+{
+    return open(UriQuery(uri));
+}
+
+RetVal<Val> MU3Launcher::open(const UriQuery& uriQuery)
+{
+    RetVal<Val> result;
+    result.ret = true;
+
+    if (uriQuery.uri().path() == "notation/measureproperties") {
+        showMeasureProperties(uriQuery.param("index").toInt());
+    } else {
+        result.ret = false;
+    }
+
+    return result;
+}
+
+ValCh<Uri> MU3Launcher::currentUri() const
+{
+    return ValCh<Uri>();
+}
+
+Ret MU3Launcher::openUrl(const std::string&)
+{
+    return false;
+}
+
+void MU3Launcher::showMeasureProperties(const int measureNumber)
```

Comment:
Tell me why we need this class if we can just create a dialog where we need it? Also, will it not turn out that there will be a bunch of methods with the creation of different dialogs?